src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,53 @@
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.nav__item {
+  margin-left: 20px;
```

Comment:
For the margin-left property, consider using the shorthand property for margin to combine the top, right, bottom, and left margins into a single property. This can make the code more concise and easier to maintain.